finance/community-updates/2022-10.txt

```diff
@@ -0,0 +1,94 @@
+Dear Astropy Community,
+
+The Finance Committee would like to give you some updates on our activities
+over the several months.
+
+
+# Cycle III Funding Results
+
+We recently completed the proposal process for allocating the next round of
+funding for projects and other major expenses. This is the first time we have
+been able to allocate money from our NASA ROSES-2020 grant as well as our
+grant from the Moore foundation.
+
+A total of 22 proposals were received, of which 20 were allocated funds.
+Although our budget didn't quite stretch to supporting all projects at their
+full requested level, every approved proposal receives at least their minimum
+funding request, and — in many cases — substantially more.
+
+The full list of proposals funded in this process is available at [1]. Each
+proposal will be allocated a dedicated Finance Committee member who will guide
+project team through the process of getting started and claiming funds.
+
+We're excited to be able to fund so many amazing projects! Thanks to those
+who submitted proposals and to everybody who took part in the discussion and
+voting process, and many congratulations to the successful project teams!
+
+The Finance Committee welcomes feedback on the funding process. Please feel
```

Comment:
```suggestion
Finally, the Finance Committee welcomes feedback on the funding process. Please feel
```
or similar.  It seems like a lot of disjointed pragraphs so a joiner seems useful